src/ducks/SANCharts/utils.js

```diff
@@ -237,7 +239,7 @@ export const generateMetricsMarkup = (
         key={`line-${dataKey}`}
         type='linear'
         yAxisId={currentYAxisId}
-        name={label}
+        name={(useShortName && shortLabel) || label}
```

Comment:
name={ use ? shortL : la } @haritonasty

Reply:
ternary here not the same case: not every metric has `shortLabel`